client/auth.go

```diff
@@ -1,29 +1,43 @@
 package client
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"net/http"
 )
 
-func (c *Client) Auth() error {
+func (c *Client) Auth() (string, error) {
```

Comment:
Does /auth return more than just id in that JSON hash? If so this function should be returning a struct.